scripts/install-deps.sh

```diff
@@ -444,14 +444,11 @@ do_install_do() {
 
     git clone --recursive --single-branch --branch $do_ref --depth 1 $do_url . || return
 
-    distro=$OS$VER
-    install_do_deps_distro="${distro//./}"
-
-    if [[ $install_do_deps_distro != "" ]]; then
-        local bootstrap_file=$do_dir/build/scripts/bootstrap.sh
-        chmod +x $bootstrap_file || return
-        $SUDO $bootstrap_file --platform "$install_do_deps_distro" --install build || return
-    fi
+    #TODO: Remove the sepecif commit once DO publishs a new tag
+    git checkout 02c9ae2c7484182903c66ad986a834762fc569e6
```

Comment:
Can we not install from the apt repository and build against that?

Reasoning for snapping to a hash?